dependency-check-maven-config.xml

```diff
@@ -192,4 +217,269 @@
+            jetty-io-9.3.14.v20161028.jar
+            jetty-jmx-9.3.14.v20161028.jar
+            jetty-servlets-9.2.19.v20160908.jar
+            jetty-util-ajax-9.2.14.v20151106.jar
+            jetty-webapp-9.3.6.v20151106.jar
+            websocket-api-9.3.6.v20151106.jar
+            ehcache-2.10.3.jar/rest-management-private-classpath/META-INF/maven/org.eclipse.jetty/jetty-io/pom.xml        </notes>
+        <cve>CVE-2017-9735</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE has to do with a command that is not used in the running system.
+            aws-java-sdk-opsworkscm-1.11.108.jar
+        </notes>
+        <cve>CVE-2015-8559</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE is out of date. The vulnerability is fixed in 3.x versions of protobuf.
+            protobuf-java-3.1.0.jar
+            pnotify-1.3.1.jar: compiler.jar/META-INF/maven/com.google.protobuf/protobuf-java/pom.xml
+        </notes>
+        <cve>CVE-2015-5237</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs only apply to the google android and google desktop libraries which do not affect the running system.
+            google-http-client-1.22.0.jar
+        </notes>
+        <cve>CVE-2016-5696</cve>
+        <cve>CVE-2014-6060</cve>
+        <cve>CVE-2014-1939</cve>
+        <cve>CVE-2013-7372</cve>
+        <cve>CVE-2010-1807</cve>
+        <cve>CVE-2007-3150</cve>
+        <cve>CVE-2007-1085</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE is disputed by SUN, but the claim is that a denial of service attack can be incurred.
+            This dependency is used for workspace email notifications and is limited by the cron string configuration.
+            This cron string configuration mitigates the possibility of a denial of service attack.
+            javax.mail-1.5.5.jar
+            mail-1.4.5.jar
+        </notes>
+        <cve>CVE-2007-6059</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE applies to ganglia-web. OWASP is flagging this CVE on gmetric4j-1.0.7.jar (as a direct transitive and as a transitive to metrics-ganglia-3.2.2.jar).
+            Though gmetric4j-1.0.7.jar has the same group id as ganglia-web, it does not depend on ganglia-web.
+            gmetric4j-1.0.7.jar
+            metrics-ganglia-3.2.2.jar
+        </notes>
+        <cve>CVE-2007-6465</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE is at the OS level and does not apply to the running system.
+            jna-4.4.0.jar
+        </notes>
+        <cve>CVE-2012-0217</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            OWASP is looking at the version of Artemis rather than the version of ActiveMQ.
+            These CVEs do not apply to the version of ActiveMQ server that is used in the system.
```

Comment:
✏️  Did we mean to write "Artemis server" instead of "ActiveMQ server" here?

Reply:
No it's meant to say ActiveMQ server, but I fixed this wording in the snippet to hopefully make it more clear. I didn't word it well the first time.